README.md

```diff
@@ -2148,24 +2158,29 @@ Which come from Cordova?
 Because startups are so much smaller than many established companies, and because they may
 grow quickly, a few other things are worth remembering when negotiating with startups:
 
-- **Cash versus equity**: If your risk tolerance is reasonably high, it’s reasonable to ask for
-  an offer with more equity and less cash.
+- **Cash versus equity**: If your
+  [risk tolerance](https://www.investopedia.com/terms/r/risktolerance.asp) is reasonably high,
+  it’s reasonable to ask for an offer with more equity and less cash.
 
   - 🚧 More discussion (perhaps in the intro to the Guide) about risk.
     What is risk? What is risk tolerance and how do I know if I have it?
     Should I be risk tolerant?
     What can I look for in the company to determine whether it’s worth a “risk”?
 
-  If a company begins to do well, it’ll likely “level up” lower salaries (bringing them
-  closer to market average) even if you got more equity up front.
-  On the other hand, if you ask for more cash and less equity, it’s unlikely you’ll be able
-  to negotiate to get more equity later on, since equity is increasingly scarce over time
-  (at least in a successful company!).
+     If a company begins to do well, it’ll likely “level up" lower salaries (bringing them
```

Comment:
Indentation looks off? Because of or because we did not Flowmark? Feel free to move the 🚧below to get it to work.